ansible/playbooks/sap-hana-preconfigure.yaml

```diff
@@ -20,14 +20,44 @@
       when: scale_out | bool
 
     # 3018133 - Linux: Running SAP applications compiled with GCC 10.x
-    - name: Ensure minimum version of GCC10 are installed
+    - name: Ensure minimum version of GCC10 are installed on SLE12
       community.general.zypper:
         name: "{{ item }}"
         state: present
+        oldpackage: yes
       loop:
-        - 'libgcc_s1>=10.2.1'
-        - 'libstdc++6>=10.2.1'
-        - 'libatomic1>=10.2.1'
+        - 'libgcc_s1=10.2.1+git583-1.3.5'
+        - 'libstdc++6=10.2.1+git583-1.3.5'
+        - 'libatomic1=10.2.1+git583-1.3.5'
+      when:
+        - ansible_facts['distribution_version'] == '12.4' or
+          ansible_facts['distribution_version'] == '12.5'
+
+    - name: Ensure minimum version of GCC10 are installed on SLE 15-SP[123]
+      community.general.zypper:
+        name: "{{ item }}"
+        state: present
+        oldpackage: yes
```

Comment:
https://docs.ansible.com/ansible/latest/collections/community/general/zypper_module.html#parameter-oldpackage

Reply:
Is probably not needed, but newer package can be installed and then it will be required.